common/src/test/scala/uk/ac/wellcome/models/MergedSierraRecordTest.scala

```diff
@@ -7,238 +7,276 @@ import uk.ac.wellcome.utils.JsonUtil
+      MergedSierraRecord(id = "111")
+    }
 
-  it("should allow creation from only a SierraBibRecord") {
-    val bibRecord = sierraBibRecord(id = "101")
-    val mergedRecord = MergedSierraRecord(bibRecord = bibRecord)
-    mergedRecord.id shouldEqual bibRecord.id
-    mergedRecord.maybeBibData.get shouldEqual bibRecord
-  }
+    it("should allow creation from only a SierraBibRecord") {
+      val bibRecord = sierraBibRecord(id = "101")
+      val mergedRecord = MergedSierraRecord(bibRecord = bibRecord)
+      mergedRecord.id shouldEqual bibRecord.id
+      mergedRecord.maybeBibData.get shouldEqual bibRecord
+    }
+
+    it("should allow creation from a SierraBibRecord and a version") {
+      val bibRecord = sierraBibRecord(id = "202")
+      val version = 10
+      val mergedRecord = MergedSierraRecord(
+        bibRecord = bibRecord,
+        version = version
+      )
+      mergedRecord.version shouldEqual version
+    }
 
-  it("should allow creation from a SierraBibRecord and a version") {
-    val bibRecord = sierraBibRecord(id = "202")
-    val version = 10
-    val mergedRecord = MergedSierraRecord(
-      bibRecord = bibRecord,
-      version = version
-    )
-    mergedRecord.version shouldEqual version
+    it("should be at version 1 when first created") {
+      val record = MergedSierraRecord(id = "555")
+      record.version shouldEqual 1
+    }
   }
 
-  it("should merge data from a bibRecord when empty") {
-    val record = sierraBibRecord(id = "222")
-    val originalRecord = MergedSierraRecord(id = "222")
+  describe("merging with a SierraBibRecord") {
+    it("should merge data from a bibRecord when empty") {
+      val record = sierraBibRecord(id = "222")
+      val originalRecord = MergedSierraRecord(id = "222")
 
-    val newRecord: Option[MergedSierraRecord] =
-      originalRecord.mergeBibRecord(record)
+      val newRecord: Option[MergedSierraRecord] =
+        originalRecord.mergeBibRecord(record)
 
-    newRecord.get.maybeBibData.get shouldEqual record
-  }
+      newRecord.get.maybeBibData.get shouldEqual record
+    }
 
-  it("should only merge bib records with matching ids") {
-    val record = sierraBibRecord(id = "333")
-    val originalRecord = MergedSierraRecord(id = "444")
+    it("should only merge bib records with matching ids") {
+      val record = sierraBibRecord(id = "333")
+      val originalRecord = MergedSierraRecord(id = "444")
 
-    val caught = intercept[RuntimeException] {
-      originalRecord.mergeBibRecord(record)
+      val caught = intercept[RuntimeException] {
+        originalRecord.mergeBibRecord(record)
+      }
+      caught.getMessage shouldEqual "Non-matching bib ids 333 != 444"
     }
-    caught.getMessage shouldEqual "Non-matching bib ids 333 != 444"
-  }
 
-  it("should be at version 1 when first created") {
-    val record = MergedSierraRecord(id = "555")
-    record.version shouldEqual 1
-  }
+    it("should always increment the version when mergeBibRecord is called") {
```

Comment:
I think we can omit the "shoulds" from tests safely, without the should it still makes sense and makes the descriptions less repetitive.

Reply:
e.g.

`it("should always increment the version when mergeBibRecord is called")`

vs. 

`it("always increments the version when mergeBibRecord is called")`